class NetworkProtocolException(Exception):
    def __init__(self):
        pass
